plaso/parsers/dsv_parser.py

```diff
@@ -43,6 +43,9 @@ class DSVParser(interface.FileObjectParser):
   # it can be defined here.
   QUOTE_CHAR = b'"'
 
+  # The maximum size of a single field in the parser
+  FILE_SIZE_LIMIT = csv.field_size_limit()
```

Comment:
this should probably be FIELD_SIZE_LIMIT

Reply:
Indeed